src/swarm-lang/Swarm/Language/Context.hs

```diff
@@ -1,7 +1,6 @@
 {-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE DerivingVia #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE PatternSynonyms #-}
-{-# LANGUAGE ViewPatterns #-}
 
 -- |
```

Comment:
In addition to outlining this module's contents (e.g. "Contexts with a serialization implementation that preserves sharing") perhaps also briefly motivate the existence of Contexts (e.g. Robots need to store variables for...)